codebase/web/modules/custom/danse_label_override/danse_label_override.module

```diff
@@ -0,0 +1,418 @@
+                    switch ($topic->id()) {
+                        case TopicInterface::CREATE:
+                            $label = (string) t('@verb_time "@title" is created', $args);
+                            break;
+                        case TopicInterface::UPDATE:
+                            $label = (string) t('@verb_time "@title" is updated', $args);
+                            break;
+                        case TopicInterface::DELETE:
+                            $label = (string) t('@verb_time "@title" is deleted', $args);
+                            break;
+                        case TopicInterface::PUBLISH:
+                            $label = (string) t('@verb_time "@title" is published', $args);
+                            break;
+                        case TopicInterface::UNPUBLISH:
+                            $label = (string) t('@verb_time "@title" is unpublished', $args);
+                            break;
+                        case TopicInterface::COMMENT:
+                            $label = (string) t('@verb all comments on "@title"', $args);
+                            break;
+                    }
+                } else if ($subscriptionMode == 2){ // Related Entity Subscription
+                    switch ($topic->id()) {
+                        case TopicInterface::CREATE:
+                            $label = (string) t('@verb_time content related to "@title" is created', $args);
+                            break;
+                        case TopicInterface::UPDATE:
+                            $label = (string) t('@verb_time content related to "@title" is updated', $args);
+                            break;
+                        case TopicInterface::DELETE:
+                            $label = (string) t('@verb_time content related to "@title" is deleted', $args);
+                            break;
+                        case TopicInterface::PUBLISH:
+                            $label = (string) t('@verb content related to "@title"', $args);
+                            break;
+                        case TopicInterface::UNPUBLISH:
+                            $label = (string) t('@verb_time content related to "@title" is unpublished', $args);
+                            break;
+                        case TopicInterface::COMMENT:
+                            $label = (string) t('@verb comments related to "@title"', $args);
+                            break;
+                    }
+                } else { // subscriptionMode = something other than 0 (Entity Type Subscriptions), 1 (Individual Entity Subscriptions), or 2 (Related Content Subscriptions)
+                    //Add Switch Statement here if needed
+                } // end Subscription Modes
+            }            
+            else if ($entityBundle == 'study'){ // labels for the forum Content Entity Bundle
+                $label = (string) t('@verb_darwin', $args);
```

Comment:
You need to handle all the cases 

case TopicInterface::CREATE:
case TopicInterface::UPDATE:
case TopicInterface::DELETE:
case TopicInterface::PUBLISH:
case TopicInterface::UNPUBLISH:
case TopicInterface::COMMENT:

For us, we need the label change on `UPDATE`